modules/http4s/build.sbt

```diff
@@ -4,7 +4,7 @@ name := "pureconfig-http4s"
 
 crossScalaVersions := Seq(scala212, scala213)
 
-libraryDependencies ++= Seq("org.http4s" %% "http4s-core" % "0.21.21")
+libraryDependencies ++= Seq("org.http4s" %% "http4s-core" % "1.0.0-M20")
```

Comment:
How close are we to a RC version here? I'd rather avoid milestone versions if possible.

Reply:
It is the first version compatible with cats-effect 3.0. Can't tell how close it is to RC state, because there are more changes than just cats-effect 3.0. Parts related to dependencies looks mostly complete for me, but it will need some time to publish with RC tag.

I checked the source of the module. It is not using any effect related methods. It could be possible to compile module with older version, but it will make it impossible to use http4s and cats-effect module in the same application anyway (because of dependency on incompatible versions of cats-effect). If this is ok (at least temporary), it could be downgraded.